assets/svg/icon-user.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
good idea to run this through svgo, there's quite a bit of stuff in this SVG that doesn't need to be there (Lines 3, 4, 5, 6 are all unnecessary, the element `id` attributes do nothing, `.00000` does nothing, etc.)